portfolio/src/main/webapp/script.js

```diff
@@ -178,14 +178,17 @@ function getCommentHTML(json, i) {
 }
 
 /**
-  * Returns a time of the form: June 16, 5:30.
+  * Returns a time of the form: Jun 16, 5:30.
   * @param {number} timeInmilliseconds The time in milliseconds since the 
       epoch. 
   * @return {String} The nicely-formatted string representing that time. 
   */
 function prettyPrintTime(timeInMilliseconds) {
   let time = new Date(0);
   time.setUTCMilliseconds(timeInMilliseconds);
+
+  // Example output of toString(): 
+  // Tue Aug 19 1975 23:15:30 GMT+0200 (CEST)
   let dateInformation = time.toString().split(" ");
```

Comment:
i think there are more optimizations that could be done here, but i'll leave as is since this is a starter project